src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -260,12 +296,22 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions, Cance
             }
 
             _functions = functions ?? new List<FunctionMetadata>();
-            _processStartupInterval = workerConfig.CountOptions.ProcessStartupInterval;
-            _restartWait = workerConfig.CountOptions.ProcessRestartInterval;
-            _shutdownTimeout = workerConfig.CountOptions.ProcessShutdownTimeout;
-            ErrorEventsThreshold = 3 * (await _maxProcessCount.Value);
 
-            if (Utility.IsSupportedRuntime(_workerRuntime, _workerConfigs))
+            if (_environment.IsMultiLanguageRuntimeEnvironment())
```

Comment:
Can we cache this in something like a private `_isMultiLanguageRuntimeEnvironment` field in the constructor? While not super-expensive, it does call `ToLower()` and `Contains()` every time it's called... would be better to cache that if it's being re-checked a lot of places in here.